porting/npl/riot/include/nimble/nimble_npl_os.h

```diff
@@ -100,7 +100,10 @@ ble_npl_eventq_get(struct ble_npl_eventq *evq, ble_npl_time_t tmo)
     } else if (tmo == BLE_NPL_TIME_FOREVER) {
         return (struct ble_npl_event *)event_wait(&evq->q);
     } else {
-        return (struct ble_npl_event *)event_wait_timeout(&evq->q, (tmo * 1000));
+        uint64_t tmo_us64 = tmo * US_PER_MS;
+        assert(tmo_us64 <= UINT32_MAX);
+        return (struct ble_npl_event *)event_wait_timeout(&evq->q,
+                                                          (tmo_us64));
```

Comment:
in the current state, `tmo_us64` should be explicitly casted to `uint32_t` I'd say

Reply:
or we simply wait for https://github.com/RIOT-OS/RIOT/pull/13373 to be merged :-)

Reply:
Waited for RIOT-OS/RIOT#13373 and modified to `event_wait_timeout64`